File: 2583/2583.c
int kthLargestSumInABinaryTree(struct TreeNode* root) {
    if (!root) return 0;
    int l = kthLargestSumInABinaryTree(root->left);
    int r = kthLargestSumInABinaryTree(root->right);
    return 1 + (l > r ? l : r);
}

File: 2583/2583.cs
public class Solution {
    public int KthLargestSumInABinaryTree(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(KthLargestSumInABinaryTree(root.left), KthLargestSumInABinaryTree(root.right));
    }
}